diff --git a/CMakeLists.txt b/CMakeLists.txt
index 1fdee53..1bdfedd 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -54,7 +54,7 @@ externalproject_add(libgmp
 
 externalproject_add(libmpfr
   DEPENDS libgmp
-  URL https://www.mpfr.org/mpfr-current/mpfr-4.1.0.tar.bz2
+  URL https://www.mpfr.org/mpfr-4.1.0/mpfr-4.1.0.tar.bz2
   URL_MD5 44b892bc5a45bafb4294d134e13aad1d
   DOWNLOAD_DIR ${CMAKE_CURRENT_BINARY_DIR}/external-downloads
   BINARY_DIR ${CMAKE_CURRENT_BINARY_DIR}/external/mpfr
